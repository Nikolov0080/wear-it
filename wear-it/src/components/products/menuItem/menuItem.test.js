import React from 'react';
import renderer from 'react-test-renderer';
import MenuItem from './menuItem';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Menu Item', () => {
    const tree = renderer
        .create(
        <BrowserRouter>
        <MenuItem
        className='active'
        index='54'
        type='good'
        image='ho ho ho test'
        />
        </BrowserRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});