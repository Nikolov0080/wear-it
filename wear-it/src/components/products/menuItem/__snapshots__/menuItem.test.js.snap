// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Menu Item 1`] = `
<div>
  <div
    className="categoryBox"
    onClick={[Function]}
  >
    <div
      className="leftBoxCategory"
    >
      <div
        className="active"
      >
        good
      </div>
    </div>
    <div
      className="rightBoxImage"
    >
      <img
        alt="category source"
        className="image"
        src="ho ho ho test"
      />
    </div>
  </div>
</div>
`;
